cli/command/cluster/add.go

```diff
@@ -134,6 +140,15 @@ func checkTopology(curveadm *cli.CurveAdm, data string, options addOptions) erro
 	return pb.Run()
 }
 
+func checkDeployType(deployType string) error {
+	for _, t := range SUPPORTED_DEPLOY_TYPES {
+		if t == deployType {
+			return nil
+		}
+	}
+	return errno.ERR_UNSUPPORT_DEPLOY_TYPE.F("deploy type: %s", deployType)
+}
+
```

Comment:
The step can place PreRunE hook function.

Reply:
I see

Reply:
I noticed that before PreRunE hook, an SQL query will happened, then it will throw error because of the mismatch.
So I think this check must be run before it.